src/Provider/LocationIQ/LocationIQ.php

```diff
@@ -43,13 +51,17 @@ final class LocationIQ extends AbstractHttpProvider implements Provider
      * @param HttpClient $client an HTTP adapter
      * @param string     $apiKey an API key
      */
-    public function __construct(HttpClient $client, string $apiKey)
+    public function __construct(HttpClient $client, string $apiKey, string $region = null)
     {
         if (empty($apiKey)) {
             throw new InvalidCredentials('No API key provided.');
         }
 
         $this->apiKey = $apiKey;
+        if (!$region || !in_array($region, $this->regions)) {
```

Comment:
```suggestion
        if (null === $region || true !== in_array($region, $this->regions, true)) {
```

I would also throw an error in the region is not valid (instead of using `us1`).